ops/terraform/env/mgmt/ecr.tf

```
@@ -1,4 +1,9 @@
 locals {
+  #Defined Admin Roles with Access to perform destructive actions on the repository
+  admin_roles_arn = toset([
```

```suggestion
  admin_role_arns = toset([
```

nit: Make this variable plural.